erizo_controller/erizoJS/models/Connection.js

```diff
@@ -0,0 +1,104 @@
+/*global require, exports*/
+'use strict';
+var addon = require('./../../../erizoAPI/build/Release/addon');
+var logger = require('./../../common/logger').logger;
+
+var log = logger.getLogger('Connection');
+
+
+
+class Connection {
+  constructor (id, threadPool, ioThreadPool) {
+    log.info(`message: constructor, id: ${id}`);
+    this.id = id;
+    this.threadPool = threadPool;
+    this.ioThreadPool = ioThreadPool;
+    this.mediaConfiguration = 'default';
+    //  {id: stream}
+    this.mediaStreams = new Map();
+    this.wrtc = this._createWrtc();
+  }
+
+  _getMediaConfiguration(mediaConfiguration = 'default') {
+    if (global.mediaConfig && global.mediaConfig.codecConfigurations) {
+      if (global.mediaConfig.codecConfigurations[mediaConfiguration]) {
+        return JSON.stringify(global.mediaConfig.codecConfigurations[mediaConfiguration]);
+      } else if (global.mediaConfig.codecConfigurations.default) {
+        return JSON.stringify(global.mediaConfig.codecConfigurations.default);
+      } else {
+        log.warn(
+          'message: Bad media config file. You need to specify a default codecConfiguration.'
+         );
+        return JSON.stringify({});
+      }
+    } else {
+      log.warn(
+        'message: Bad media config file. You need to specify a default codecConfiguration.'
+      );
+      return JSON.stringify({});
+    }
+  }
+
+  _createWrtc() {
+    var wrtc = new addon.WebRtcConnection(this.threadPool, this.ioThreadPool, this.id,
+      global.config.erizo.stunserver,
+      global.config.erizo.stunport,
+      global.config.erizo.minport,
+      global.config.erizo.maxport,
+      false,
+      this._getMediaConfiguration(this.mediaConfiguration),
+      global.config.erizo.useNicer,
+      global.config.erizo.turnserver,
+      global.config.erizo.turnport,
+      global.config.erizo.turnusername,
+      global.config.erizo.turnpass,
+      global.config.erizo.networkinterface);
+
+    return wrtc;
+  }
+
+  _createMediaStream (id) {
+    log.debug(`message: _createMediaStream, connectionId: ${this.id}, mediaStreamId: ${id}`);
+    const mediaStream = new addon.MediaStream(this.threadPool, this.wrtc, id,
+      this._getMediaConfiguration(this.mediaConfiguration));
+    mediaStream.id = id;
```

Comment:
Do you mean in erizo+erizoAPI? If that's the case, that would make sense. We could also do that with connection and WebRTCConnection and always get the id from the c++ counterpart instead of the JS representation.
I prefer doing that in another PR (to avoid further complicating this one with c++ code) and do it consistently for both. What do you think?